node/src/isolate/external.cpp

```diff
@@ -0,0 +1,200 @@
+#include "cocaine/detail/isolate/external.hpp"
+#include "cocaine/idl/isolate.hpp"
+
+#include <cocaine/context.hpp>
+#include <cocaine/dynamic.hpp>
+#include <cocaine/errors.hpp>
+#include <cocaine/logging.hpp>
+#include <cocaine/engine.hpp>
+
+#include <cocaine/rpc/session.hpp>
+
+#include <cocaine/traits/dynamic.hpp>
+
+#include <blackhole/logger.hpp>
+#include <asio/ip/tcp.hpp>
+
+using asio::ip::tcp;
```

Comment:
I'm not sure if it really does create symbols ::tcp::* or it's just language feature for name resolving (I've always thought that it's just about resolving, not creating extra symbols)